enclave-manager/api/protobuf/kurtosis_enclave_manager_api.proto

```diff
@@ -55,3 +56,8 @@ message InspectFilesArtifactContentsRequest {
   int32 apic_port = 2;
   api_container_api.FilesArtifactNameAndUuid file_names_and_uuid = 3;
 }
+
+message GetStarlarkRunRequest{
+  string apic_ip_address = 1;
+  int32 apic_port = 2;
```

Comment:
why not a `uint32` since ports are nonnegative?

Reply:
that's a good point! It should have been. We'll need to change this in the other requests too.